src/styles/book.css

```diff
@@ -0,0 +1,125 @@
+.d-flex {
+  display: flex;
+}
+
+.book-container {
+  font-family: 'Roboto Slab', serif;
+  justify-content: space-between;
+  align-items: center;
+  border: solid 1px rgb(214, 214, 214);
+  border-radius: 4px;
+  margin-left: -41px;
+  margin-top: 10px;
+  height: 7rem;
+  padding: 10px 5%;
+  background-color: #fff;
+}
+
+.general-info {
+  flex-flow: column wrap;
+  width: 30%;
+}
+
+.title {
+  font-size: 1.375rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+  color: #121212;
+  margin: 0;
+}
+
+.book-management {
+  list-style-type: none;
+  padding: 0;
+}
+
+.current-chapter > button {
+  width: 200px;
+  height: 2.063rem;
+  margin-top: 1.6rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  background-color: #0290ff;
+  color: azure;
+  border: none;
+  border-radius: 3px;
+  font-size: 0.813rem;
+  letter-spacing: 0.5px;
+  cursor: pointer;
+}
+
+.book-management > li > button {
+  margin: 0 15px 0 0;
+  border: none;
+  cursor: pointer;
+  padding: 0;
+  font-family: 'Robotoslab', sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
```

Comment:
- [ ] It will be better if you add `background: #fff` to the button to make it match the design in the zipline.